Source/DafnyServer/CounterExampleGeneration/DafnyModelVariable.cs

```diff
@@ -6,6 +6,8 @@
 using System.Linq;
 using System.Text.RegularExpressions;
 using Microsoft.Boogie;
+using Microsoft.Dafny;
```

Comment:
Can you please fix these nullable annotations warnings above that the CI raised?

Reply:
Have added `#nullable enable` to fix this